components/massmint/OnBoarding.vue

```diff
@@ -51,6 +56,17 @@
                 descriptionTabs[activeDescriptionTab].fileStructureDescription
               "
               class="fixed-height white-space-break-spaces-mobile code" />
+            <div class="is-flex is-justify-content-flex-end mt-2">
+              <NeoButton
+                v-safe-href="
+                  `/massmint/template.${activeDescriptionTab.toLowerCase()}`
+                "
+                :label="$t('massmint.onboarding.downloadTemplate')"
+                variant="text"
+                no-shadow
+                tag="a"
```

Comment:
`nuxt-link`

Reply:
`<a/>` tag with the attribute `download` could make the download functionality work.